src/Ses/Input/SendEmailRequest.php

```diff
@@ -222,12 +231,15 @@ public function requestBody(): string
             })($this->Content);
         }
 
-        (static function (array $input) use (&$payload) {
+        (static function (array $input) use (&$payload, $indices) {
+            $indices->k4a82301 = -1;
             foreach ($input as $value) {
+                ++$indices->k4a82301;
+
                 if (null !== $value) {
-                    (static function (MessageTag $input) use (&$payload) {
-                        $payload['EmailTags']['member'][]['Name'] = $input->getName();
-                        $payload['EmailTags']['member'][]['Value'] = $input->getValue();
+                    (static function (MessageTag $input) use (&$payload, $indices) {
+                        $payload['EmailTags'][$indices->k4a82301]['Name'] = $input->getName();
+                        $payload['EmailTags'][$indices->k4a82301]['Value'] = $input->getValue();
```

Comment:
This is why I introduced `indices` in that serializer.

The previous code geneates 1 item per property (because of `[]`) 
```
$payload['foo'][]['Key1'] = '1';
$payload['foo'][]['Key2'] = '2';
```